print-battleground-state-changes

```diff
@@ -265,7 +265,9 @@ for (state, timestamped_results) in summarized.items():
     print(f'\n{state}:')
     print(tabulate([string_summary(summary) for summary in timestamped_results]))
 
-    html_chunks.append("<table class='table table-bordered'>")
+    # TODO: Use a better slug for "North Carolina (15)" (will just be "north" for now)
```

Comment:
If we just do `split("(")` that should work. Then replace the " " with a dash or something

Reply:
Good point, on it ...